### PR TITLE
Translate completely everything. From the notes to the messages in the russian translation

### PR DESCRIPTION
**Pull request by thebashforkbomber**
by the comparision you think that there are errors, but that is infact just github displaying it wrongly

### CODE REVIEW

**Comment by thebashforkbomber** [comment]
the current translation has inperfections (one message has the word TPA, the other related to the first message doesn't)
and it isn't the only one

File: Russian/config.yml [hunk 1]
```diff
@@ -29,89 +29,83 @@ TpaToggle-Deactivated-Player-2: "&c%player% &6отключил &ctpa&6 для т
...
 Deny-Message: "&6Для отмены телепорта введи &c/tpdeny"
-Sent-Request-On-You: "&c%player% &6отправил вам запрос на телепорт!"
+Sent-Request-On-You: "&cигрок &6отправил вам запрос на телепорт!"
 Timed-Out-Here-Message: "&6Ваш запрос на телепорт сюда &cистек!"
-Sending-Teleport-Here-Request: "&6Отправка запроса на телепорт к &c%player%&6. Для отмены запроса ввведи &c/tpcancel player"
-Sent-Here-Request-On-You: "&c%player% &6Отправил вам запрос на телепорт!"
-Already-A-TPHere-Request: "&6Вы уже &cотправляли &6запрос на телепортацию! Для отмены введите в чате &c/tpcancel player"
-Tpa-Here-Usage-Message: "&6Используй: &c/tpahere (player)"
+Sending-Teleport-Here-Request: "&6Отправка запроса на телепорт к &c%player%&6. Для отмены запроса введи &c/tpcancel игрок"
+Sent-Here-Request-On-You: "&cигрок &6Отправил вам запрос на телепорт!"
```
Comment (kit-katkittycat) [inline Russian/config.yml:45]: Alright this might look weird, ik, but this later turns into %player%

**Review by MuleMuleDupe — APPROVED** [review]
Approved! Let me know your discord account to add you to the translator roles! Have great holidays!

**Comment by MuleMuleDupe** [comment]
Actually contact me in the discord https://discord.com/invite/SZUkza9

**Comment by thebashforkbomber** [comment]
> Actually contact me in the discord https://discord.com/invite/SZUkza9

how tho

**Comment by thebashforkbomber** [comment]
> Approved! Let me know your discord account to add you to the translator roles! Have great holidays!

username: vcp.ru.en2058

**Pull request merged**